CONTRIBUTING.md

```diff
@@ -215,11 +215,14 @@ Per the
 [API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter),
 `get_()` prefixes are discouraged.
 
-#### Alphabetize enum variants
+#### Enum variants
 
 When implementing or modifying an `enum` type, list its variants in alphabetical
 order.
 
+Prefer active verbse for variant names. E.g. `Allow` instead of `Allowed`,
```

Comment:
"verbse". The other thing I noticed recently which I consider unidiomatic is faux-bools like `::Yes` and `::No` -- instead, use variant names that are descriptive of the different states.

Reply:
> "verbse".

Fixed.

> The other thing I noticed recently which I consider unidiomatic is faux-bools like ::Yes and ::No

Explicitly mentioned this as well.